src/deepsparse/pipeline.py

```diff
@@ -308,7 +310,7 @@ def run(
                     return operator_output
 
             if next_step == self.router.START_ROUTE:
-                operator_output = run_func(
+                operator_output = self.run_func_with_middleware(
```

Comment:
just want to confirm that we are wrapping only the operator itself, not the submit call

Reply:
Should be just the callable in general. 
You mean OperatorScheduler submit? https://github.com/neuralmagic/deepsparse/blob/617a62ff7847f42e00c2ddb5a5589d09a753ed55/src/deepsparse/schedulers/scheduler.py#L66

Reply:
operator and submit call are both callables. but it seems like you're fetching the Operator from the kwarg so should be fine. Maybe add a test to verify?

Reply:
Are there existing tests for this flow ?